internal/outpost/proxyv2/application/application.go

```diff
@@ -118,8 +118,8 @@ func NewApplication(p api.ProxyOutpostConfig, c *http.Client, server Server, old
 	mux := mux.NewRouter()
 
 	// Save cookie name, based on hashed client ID
-	h := sha256.New()
-	bs := string(h.Sum([]byte(*p.ClientId)))
+	h := sha256.Sum256([]byte(*p.ClientId))
+	bs := string(h[:])
```

Comment:
Does this break existing sessions? Or is the resulting hash the same?

Reply:
it does change the name of the cookie as with the previous invocation we were basically not hashing the value at all @rissson